### ISSUE TITLE
Support for hidden warnings and tags

### ISSUE THREAD

**Issue by jdm** [LazyCats-dev/ao3-podfic-posting-helper#8]
jocsykes reported that https://archiveofourown.org/works/1129500 threw an error: https://cdn.discordapp.com/attachments/750437912729878578/853306644863516742/Screenshot_2021-06-12_at_17.15.27.png

**Comment by jdm** [comment]
That's https://github.com/LazyCats-dev/ao3-podfic-posting-helper/blob/145a1de18e1f336469c87500c8d0c0f386d14b13/src/inject.js#L330

**Comment by jdm** [comment]
I haven't been able to reproduce the problem in either Firefox or Chrome.

**Comment by jermowery** [comment]
Following up on discord

The error indicates that our script can't find a checkbox on the new work page

**Comment by jermowery** [comment]
We figured out what the issue was, the user had warnings hidden which is a setting we propagated during the fetch to import. As a more graceful failure mode I made the extension alert the user when the work could only be partially imported.

Long term though, we should figure out if we can support either of these options. During my initial investigations I discovered that the warnings and tags are not in the DOM when hidden so we can't just find them elsewhere. We could omit credentials when doing the fetch but that also means we wouldn't have access to works that only logged in users can see
